test-data/unit/check-functions.test

```diff
@@ -2302,3 +2302,14 @@ f = lambda: 5
 reveal_type(f)
 [out]
 main:2: error: Revealed type is 'def () -> builtins.int'
+
+[case testRevealLocalsFunction]
```

Comment:
Thanks for adding tests! I have one idea for additional test: using `reveal_locals()` inside class body.

Reply:
Is the test `testRevealLocalsClass` sufficient for this?